client/client-multi/client-objective-c/Kaa/KaaClientFactory.h

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
This issue is in all Objective-C SDK source files, so I created separate ticket for this http://jira.kaaproject.org/browse/KAA-1018